front/src/views/EnableStandardProduct/components/EnableStandardProductForm.tsx

```diff
@@ -0,0 +1,249 @@
+import {
+  HStack,
+  Stack,
+  Button,
+  Box,
+  Text,
+  Skeleton,
+  Select,
+  FormControl,
+  FormLabel,
+  VStack,
+  Input,
+} from "@chakra-ui/react";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { AlertWithoutAction } from "components/Alerts";
+import { NewNumberField } from "components/Form/NumberField";
+import SelectField from "components/Form/SelectField";
+import SwitchField from "components/Form/SwitchField";
+import { useAtomValue } from "jotai";
+import { useEffect, useMemo } from "react";
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+import { selectedBaseAtom } from "stores/globalPreferenceStore";
+import { z } from "zod";
+
+const SingleSelectOptionSchema = z.object({
+  label: z.string(),
+  value: z.string(),
+});
+
+export const EnableStandardProductFormSchema = z.object({
+  // see BoxEdit.tsx how to validate a select field
+  standardProduct: SingleSelectOptionSchema.refine(Boolean, {
+    message: "Please select a standard product",
+  }).transform((selectedOption) => selectedOption || z.NEVER),
+  category: SingleSelectOptionSchema.optional(),
+  gender: z.string().optional(),
+  sizeRange: SingleSelectOptionSchema.optional(),
+  comment: z
+    .string()
+    .optional()
+    .transform((value) => (value === "" ? undefined : value)),
+  inShop: z.boolean().optional(),
+  price: z
```

Comment:
@HaGuesto JS allows a max value of 2**53 for integers but the graphql spec only allows up to 2**31-1: https://spec.graphql.org/October2021/#sec-Int; adding a lower-than validation here would help to avoid throwing an error in the BE